app/locales/en/translations.js

```diff
@@ -1278,6 +1278,27 @@ export default {
         appointmentDate: 'Date'
       }
     },
+    labs: {
```

Comment:
Hey, I'm interested in helping out with the translations. Wondering why `labs` is pluralized when `pricing` is singular?